azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageListView.kt

```diff
@@ -190,7 +189,11 @@ internal fun PreviewMessageListView() {
         MessageListView(
             showLoading = false,
             modifier = Modifier.padding(0.dp),
-            messages = MOCK_MESSAGES.toViewModelList(LocalContext.current, MOCK_LOCAL_USER_ID),
+            messages = MOCK_MESSAGES.toViewModelList(
+                context = LocalContext.current,
+                localUserIdentifier = MOCK_LOCAL_USER_ID,
+                hiddenParticipant = mutableSetOf()
```

Comment:
should this be hiddenParticipants instead of hiddenParticipant?

Reply:
Yes Leah, I will fix this as part of refactoring.